public/js/new-user.js

```diff
@@ -0,0 +1,18 @@
+let users = [];
+
+class NewUser {
+  constructor(username){
+    this.username = username;
+    users.push(username)
+  }
+
+  isValid() { //should edit
```

Comment:
Awesome! Though instead of calling seperate functions, this isValid function should probably just return true or false, then the successUserName function will check whether the isValid function has returned true or false